src/main/modules/nunjucks/index.ts

```diff
@@ -20,6 +20,22 @@ export class Nunjucks {
       return typeof prop === 'function' ? prop(this.ctx) : prop;
     });
 
+    env.addGlobal('getError', function (fieldName: string): { text?: string } | boolean {
+      const { form, sessionErrors, errors } = this.ctx;
+
+      const hasMoreThanTwoFields = Object.keys(form.fields).length >= 2;
+      if (!hasMoreThanTwoFields || !sessionErrors.length) {
+        return false;
+      }
+
+      const fieldError = sessionErrors.find(error => error.propertyName === fieldName);
+      if (!fieldError) {
+        return false;
+      }
+
+      return { text: errors[fieldName][fieldError.errorType] };
```

Comment:
should this not be sessionErrors?

not sure what errors is.

Reply:
On line 31 I'm checking to see if this field has an error using `sessionErrors`. So technically I am getting the error from `sessionErrors` for each field.

Reply:
just confused on what errors object is as we only return sessionErrors in the getController

Reply:
`errors` is a language object is from the [`content.ts:12`](https://github.com/hmcts/nfdiv-frontend/pull/340/files#diff-945fc05018a3db2c23dab2c87c47229550f445c57676fdcd83cd8058471e8f6dR12), which contains the two error messages for the fields.

And according to TypeScript `sessionErrors` is:

```ts
type FormError = {
  propertyName: string;
  errorType: string;
};

type sessionErrors = FormError[] | undefined;
```

So I'm finding the field based on `propertyName === fieldName` and then using the `errorType` to get the corresponding field error message from it's content file.

Reply:
ahh, forgot about errors object in content. all good